src/main/java/net/greghaines/jesque/worker/WorkerImpl.java

```diff
@@ -541,7 +547,7 @@ protected void recoverFromException(final String curQueue, final Exception ex) {
                 end(false);
             } else {
                 authenticateAndSelectDB();
-                LOG.info("Reconnected to Redis");
+                LOG.info("Reconnected to Redis " + this);
```

Comment:
Is this have toString?
How will this print look like?

Reply:
This way 

18:52:24.922 [Worker-6 Jesque-DEVELOPMENT: RUNNING] INFO net.greghaines.jesque.worker.WorkerImpl - Reconnected to Redis resque:worker:dimav-e7470:23990-6:JAVA_DYNAMIC_QUEUES,highPriorityQueue,lowPriorityQueue
18:52:24.983 [Worker-10 Jesque-DEVELOPMENT: RUNNING] INFO net.greghaines.jesque.worker.WorkerImpl - Reconnected to Redis resque:worker:dimav-e7470:23990-10:JAVA_DYNAMIC_QUEUES,highPriorityQueue,lowPriorityQueue

instead of 

18:52:24.922 [Worker-6 Jesque-DEVELOPMENT: RUNNING] INFO net.greghaines.jesque.worker.WorkerImpl - Reconnected to Redis 
18:52:24.983 [Worker-10 Jesque-DEVELOPMENT: RUNNING] INFO net.greghaines.jesque.worker.WorkerImpl - Reconnected to Redis 


Probably you are right , as we have specific worker related information in the thread name :-)

Reply:
What bothered me is that the message says Redis and than we print the worker. Also believe that the worker is not really relevant so expected the Redis to be printed.

Reply:
I wanted to improve original message and now I just want to revert my change and leave the message intact.

Reply:
BTW , worker is relevant ,as every worker contains dedicated jedis client and we are reconnecting this jedis to restarted redis